.gitignore

```diff
@@ -58,6 +58,10 @@ packages/buildinfo/src/buildinfo.ts
 
 *.jwt
 
+# playwright output
+test/test-remote/uishot-*
```

Comment:
Would it maybe make sense to have a screenshot directory in `test-remote` that can be ignored?

Reply:
Agree. Although, I'm trying to keep my structure changes to a minimium with this PR though so have not changed how screenshots work or are saved. I put this in so I didn't accidently commit them.